src/domain/usecases/ImportTemplateUseCase.ts

```diff
@@ -330,7 +330,7 @@ export const compareDataPackages = (
 
 const formatDhis2Value = (item: DataPackageDataValue, dataForm: DataForm): DataPackageDataValue | undefined => {
     const dataElement = dataForm.dataElements.find(({ id }) => item.dataElement === id);
-    const booleanValue = String(item.value) === "true" || item.value === "Yes";
+    const booleanValue = String(item.optionId) === "true" || item.optionId === "true";
```

Comment:
I'd need some time to test this thoroughly in a real import/export (let me know if I should take the time, I think you guys already tested it), but only: isn't the second condition in the OR redundant?

Reply:
I tested it with @ifoche very briefly and it works but I will let @ifoche answer if you should test it in more detail. Regarding your question, probably @idelcano has a better understanding of the solution and can answer it...

Reply:
HI @tokland I think I tried all the combinations (import/export in English, import/export in Portuguese, and check values)
although it is a bit critical change because without this last change the boolean values ​​were being erased when you imported in portuguese, maybe an extra test is always there
well.


About the redundant code, I really don't know if it's necessary, it comes from alexis, speaking with nacho he doesn't see a problem in leaving it.

Reply:
Fair enough, let's keep the code that we know works. I have no comments about the i18n part, I'll approve it.